Source/relay/TourGuide/Items of the Month/Superhero Cape.ash

```diff
@@ -22,32 +29,32 @@ void IOTMSuperheroCapeGenerateResource(ChecklistEntry [int] resource_entries)
     CapeConfiguration [string] cape_configurations = {
         "vampire":new CapeConfiguration(
             "Vampire Slicer",
-            "Muscle +30% Maximum HP +50",
+            "+30% muscle, +50 HP",
             string [string] {
                 "hold":"+3 all resistances",
-                "thrill":"+3 Muscle stats per fight",
-                "kiss":"Skill: Smooch of the Daywalker<br>Drain 20% of your foe's HP",
-                "kill":"Skill: Slay the Dead<br>Instakill (costs turn, <font color='red'>Requires sword.</font>)<br>Reduces evil in the The Cyrpt zones by 1%."
+                "thrill":"+3 muscle stats/fight",
+                "kiss":"Skill: Smooch of the Daywalker|Drain 20% of your foe's HP",
+                "kill":"Skill: Slay the Dead|Undead insta-bone (NOT free kill, <font color='red'>requires sword</font>).|+1% Cyrpt elivness recdution."
```

Comment:
"elivness recdution" - is that on purpose?

Reply:
mispselled cemetery